egg/zoo/compo_vs_generalization/train.py

```diff
@@ -163,16 +170,6 @@ def main(params):
         train, test_data = split_train_test(full_data, 0.2)
         test, validation = split_train_test(test_data, 0.5)
 
-        train, test, validation, full_data = [
-            one_hotify(x, opts.n_attributes, opts.n_values)
-            for x in [train, test, validation, full_data]
-        ]
-
-        train, validation = ScaledDataset(train, 1), ScaledDataset(validation, 1)
-        test, full_data = ScaledDataset(test, 1), ScaledDataset(full_data, 1)
-        full_data_loader = DataLoader(full_data, batch_size=opts.batch_size)
-        print(" - done")
```

Comment:
Any specific reason for removing this?

Reply:
I just did some code reorganizing here to prevent a useless repeat